wsdl/ver10/schema/metadatastream.xsd

```diff
@@ -389,6 +389,34 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 		</xs:attribute>
 		<xs:anyAttribute processContents="lax"/>
 	</xs:complexType>
+	<xs:complexType name="ObjectSnapshot">
+		<xs:sequence>
+			<xs:element name="ImageRef" type="xs:anyURI" minOccurs="0"/>
+			<xs:element name="Image" type="xs:base64Binary" minOccurs="0"/>
+			<xs:element name="BoundingBox" type="tt:Rectangle" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Bounding box definition in normalized coordinates.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>      <!-- first ONVIF then vendor --> 
+		</xs:sequence>
+		<xs:attribute name="CaptureTime" type="xs:dateTime"/>
+		<xs:anyAttribute processContents="lax"/>
+	</xs:complexType>
+	<xs:complexType name="ObjectTrackComplement">
```

Comment:
Questioning whether we should pick some properties from tt:Object. Alternative would be to just provide a number of tt:Object instances with sample time information.

Reply:
It ties into to my other comment about opening up the structure to make it generic - which makes it hard to control/filter data is consumed by the client.

Reply:
Defining a new type with sparse information from tt:Object requires vendors to request additions from ONVIF for each new feature. Agree with need for restriction to avoid that device vendors are required to implement unnecessary complexity.

Propose to use something like xpath or topic filter expression. Device can signal via options what they support. A generic approach like this avoids that for each new parameter a definition by ONVIF is required.